packages/syspurpose/Pipfile

```diff
@@ -4,6 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [dev-packages]
+build_ext = { path = "../build_ext", editable = true }
 
 [packages]
```

Comment:
Syspurpose requires now package 'six'. Please add here something like this:

```
six = "*"
```

Reply:
I added it under `packages`.

Reply:
Hi, you added it under `[requires]`, but it should be under `[packages]`. Otherwise you have to run `pipenv install six`.